frontend/src/components/datasets/DatasetsListContent.js

```diff
@@ -0,0 +1,123 @@
+import { Button, Checkbox, Select, Spin, Switch } from "antd";
+const { Option } = Select;
+import React, { useCallback } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import {setReleaseFlags, listTable, setFilter, setFilterOption, clearFilters, setSortBy} from "../../modules/datasets/actions";
+import AppPageHeader from "../AppPageHeader";
+import { DATASET_FILTERS } from "../filters/descriptions";
+import FiltersWarning from "../filters/FiltersWarning";
+import getFilterProps from "../filters/getFilterProps";
+import getNFilters from "../filters/getNFilters";
+import PageContent from "../PageContent";
+import PaginatedTable from "../PaginatedTable";
+import moment from "moment";
+
+const getTableColumns = () => {
+    return [
+        {
+            title: "ID",
+            dataIndex: "id",
+            sorter: true,
+            width: 90,
+            render: (id, _) => {
+                return <Link to={`/datasets/${id}`}>
+                <div>{id}</div>
+              </Link>
+            }
+        },
+        {
+            title: "Run",
+            dataIndex: "run_name",
+            sorter: true,
+        },
+        {
+            title: "Project",
+            dataIndex: "project_name",
+            sorter: true,
+        },
+        {
+            title: "Lane",
+            dataIndex: "lane",
+            sorter: true,
+        },
+        {
+            title: "Files Released",
+            dataIndex: "release_flag_count",
+            sorter: true,
+            render: (release_flag_count, dataset) => {
+                return `${release_flag_count}/${dataset?.files?.length}`
+            }
+        },
+        {
+            title: "Last Release Time",
```

Comment:
I would rename this Release Date and format it to only display the date part.
Also, In order to give an indication that the whole dataset is released (state either Release or Block no remaining Available) i would only display this date if there is no more Available (or whatever the default name we give) file in the dataset. That would give an indication there is no more action to be done for this dataset (in case someone forget to release a file for some reason).